### ISSUE TITLE
Add meta data ? 

### ISSUE THREAD

**Issue by SerpentDrago** [iwannabelikemike/plugin.video.sparkle#9]
https://www.thesportsdb.com/

**Comment by drinfernoo** [comment]
Yes, this would be sweet, especially if team artwork and stuff could be added.

**Comment by drinfernoo** [comment]
Wow, I actually found that TheSportsDB has a Kodi module for all it's API calls... https://github.com/enen92/script.module.thesportsdb. That could be useful 😉
